### PR TITLE
🌍 : Update German Translations

### PR DESCRIPTION
**Pull request by marlonka**
## Summary
I used the new english translation as a base for additional and improved german translations

## Change Type
- [x] Translation update

### CODE REVIEW

File: client/src/localization/languages/De.ts [hunk 1]
```diff
@@ -325,8 +422,27 @@ export default {
...
+  com_nav_lang_japanese: '日本語',
+  com_nav_lang_swedish: 'Svenska',
+  com_nav_lang_korean: '한국어',
+  com_nav_lang_vietnamese: 'Tiếng Việt',
+  com_nav_lang_traditionalchinese: '繁體中文',
+  com_nav_lang_arabic: 'العربية',
+  com_nav_lang_turkish: 'Türkçe',
+  com_nav_lang_dutch: 'Nederlands',
+  com_nav_lang_indonesia: 'Indonesia',
+  com_nav_lang_hebrew: 'עברית',
```
Comment (berry-13) [inline client/src/localization/languages/De.ts]: Would you mind removing these translations?

**Review by berry-13 — CHANGES_REQUESTED** [review]

File: client/src/localization/languages/De.ts (hunk 1 above)
Comment (marlonka) [reply client/src/localization/languages/De.ts]: i removed it from my fork, sry

Comment (berry-13) [reply client/src/localization/languages/De.ts]: no problem, thanks

**Review by berry-13 — APPROVED** [review]

File: client/src/localization/languages/De.ts [hunk 2]
```diff
@@ -1,46 +1,116 @@
...
   com_ui_examples: 'Beispiele',
   com_ui_new_chat: 'Neuer Chat',
-  com_ui_happy_birthday: 'Es ist mein erster Geburtstag!',
+  com_ui_happy_birthday: 'Es ist mein 1. Geburtstag!',
   com_ui_example_quantum_computing: 'Erkläre Quantencomputing in einfachen Worten',
-  com_ui_example_10_year_old_b_day:
-    'Hast du eine kreative Idee für den Geburtstag eines 10-Jährigen?',
-  com_ui_example_http_in_js: 'Wie stelle ich eine HTTP-Anfrage in Javascript?',
+  com_ui_example_10_year_old_b_day: 'Hast du eine kreative Idee für den Geburtstag eines 10-Jährigen?',
+  com_ui_example_http_in_js: "Wie stelle ich eine HTTP-Anfrage in Javascript?",
```
Comment (berry-13) [inline client/src/localization/languages/De.ts]: use single quote

File: client/src/localization/languages/De.ts [hunk 3]
```diff
@@ -150,70 +236,73 @@ export default {
...
-    'Setzt benutzerdefinierte Anweisungen, die in die System Nachricht aufgenommen werden. Standardmäßig: keine',
+    'Setzt benutzerdefinierte Anweisungen, die in die Systemaufforderung aufgenommen werden. Standard: keine',
   com_endpoint_import: 'Importieren',
-  com_endpoint_set_custom_name:
-    'Lege einen benutzerdefinierten Namen fest, für den Fall, dass du diese Vorgabe finden kannst',
+  com_endpoint_set_custom_name: 'Lege einen benutzerdefinierten Namen fest, für den Fall, dass du diese Vorgabe finden kannst',
   com_endpoint_preset_delete_confirm: 'Bist du sicher, dass du diese Vorgabe löschen willst?',
-  com_endpoint_preset_clear_all_confirm:
-    'Bist du sicher, dass du alle deine Voreinstellungen löschen willst?',
+  com_endpoint_preset_clear_all_confirm: "Bist du sicher, dass du alle deine Voreinstellungen löschen willst?",
```
Comment (berry-13) [inline client/src/localization/languages/De.ts]: use single quote

File: client/src/localization/languages/De.ts [hunk 4]
```diff
@@ -227,72 +316,78 @@ export default {
...
   com_endpoint_config_google_service_key: 'Google Service Account Key',
   com_endpoint_config_google_cloud_platform: '(von Google Cloud Platform)',
   com_endpoint_config_google_api_key: 'Google API Key',
-  com_endpoint_config_google_gemini_api: '(Gemini API)',
-  com_endpoint_config_google_api_info:
-    'Um deinen Generative Language API-Schlüssel (für Gemini) zu erhalten,',
-  com_endpoint_config_key_import_json_key: 'Importiere Service Account JSON Key.',
-  com_endpoint_config_key_import_json_key_success:
-    'Erfolgreich importierter Service Account JSON Key',
+  com_endpoint_config_google_gemini_api: "(Gemini API)",
```
Comment (berry-13) [inline client/src/localization/languages/De.ts]: use single quote

**Review by berry-13 — CHANGES_REQUESTED** [review]
my bad, I forgot to check for lint errors

File: client/src/localization/languages/De.ts (hunk 4 above)
Comment (marlonka) [reply client/src/localization/languages/De.ts]: solved

File: client/src/localization/languages/De.ts (hunk 3 above)
Comment (marlonka) [reply client/src/localization/languages/De.ts]: solved

**Review by danny-avila — APPROVED** [review]

**Pull request merged**